server/bin/test-bin/cat

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ ! -z "${1}" && "$(basename ${1})" == "tarball-bad-prefix-file_1970.01.01T00.00.00.prefix" ]]; then
+    # The server test-20 expects a file of this name to fail when `cat` tries
+    # to read it.
+    exit 1
+fi
+/usr/bin/cat ${*} 
```

Comment:
I removed this file from my test environment and the unit tests pass with flying colors.  Can you verify why this is needed?

Reply:
See `Draft` PR #2398 CI output.

Reply:
I tried removing it, and it now seems to work without it.